microstoragetest/storage.go

```diff
@@ -0,0 +1,18 @@
+package microstoragetest
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+)
+
+// Must creates a configured storage ready to be used in tests.
+func Must() microstorage.Storage {
```

Comment:
I think the only way that makes sense here is to do: `microloggertest.Must(microloggertest.New())`. I still think this is clumsy and not necessary.

I understand `template.Must` because it may panic because of wrong user's input. This is not the case here.

If we followed that path, we should add `Must` to name of every function that accesses slice because it can panic.